com/course/zenkouv/src/main/java/lecture6/model/Contributors.java

```diff
@@ -0,0 +1,21 @@
+package lecture6.model;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import java.io.IOException;
+import java.util.List;
+
+public class Contributors {
+
+    private List<GitHubUser> contributors;
+
+    public List<GitHubUser> getContributors() {
+        return contributors;
+    }
+
+    public void castResponseToContributorList(String content) throws IOException {
```

Comment:
model should be responsible for casting response to himself

Reply:
fix